bin/xbps-query/xbps-query.1

```diff
@@ -277,6 +282,70 @@ expression wins.
 This expects an absolute path.
 This mode only works with repositories.
 .El
+.Sh FORMAT STRINGS
+Variables are package properties if not otherwise documented.
+See
+.Sx PROPERTIES
+section for a list of available properties.
+.Pp
+As example a format string like:
+.Bd -offset indent -literal
+{pkgname:<30} {installed_size!humanize :>10}\\n
```

Comment:
is `pkgname` a property? if not, would it be possible to have it (and maybe `version`?) as a "pre-filtered" pkgver?

Reply:
`pkgname` is added by xbps "dynamically" at runtime, which kinda sucks but we could probably do that for version too.

Reply:
ah i see pkgname in the manpage now

Reply:
maybe have `{pkgver!pkgname}` and `{pkgver!version}` or something like that, nut sure if this really matters, maybe I should just change this in the documentation to `pkgver`.

Reply:
well pkgname is listed in PROPERTIES, so it should be fine. adding version as a dynamic property is fairly minor imo, so could be done later.

Reply:
Main issue with the `pkgname` prop is that for the pkgdb it is added when the pkgdb is loaded, for packages out of repodata it sets it when you use `xbps_repo_get_pkg`, so this may not be set for certain cases where it iterates of the packages in the repo.

I think xtraeme added this as a hack because the code converted pkgvers to pkgnames all over the place.